bokehjs/src/lib/models/widgets/markup.ts

```diff
@@ -36,6 +48,39 @@ export abstract class MarkupView extends WidgetView {
     this.markup_el = div({class: clearfix, style})
     this.el.appendChild(this.markup_el)
   }
+
+  has_math_disabled() {
+    return this.model.disable_math || !this.contains_tex_string(this.model.text)
+  }
+
+  process_tex(): string {
+    if (this.provider.status !== "loaded") return this.model.text
```

Comment:
String equality is not needed (and not encouraged in this codebase). Also please split off `return` into a new line.